lib/Parser/Parse.cpp

```diff
@@ -11658,6 +11658,7 @@ ParseNodePtr Parser::Parse(LPCUTF8 pszSrc, size_t offset, size_t length, charcou
             m_currentNodeFunc->sxFnc.functionId = m_functionBody->GetLocalFunctionId();
             m_currentNodeFunc->sxFnc.nestedCount = m_functionBody->GetNestedCount();
             m_currentNodeFunc->sxFnc.SetStrictMode(!!this->m_fUseStrictMode);
+            m_currentNodeFunc->sxFnc.ClearFlags();
```

Comment:
This is missing initialization, right?  
What about other fields in sxFnc?
Looking at other places where we create a knopFncDecl, there are a lot more field.
Should we initialize those as well?   Should we refactor and have a "CreateFncDeclNode" to initialize them all?